.github/workflows/ci.yml

```
@@ -1,17 +1,17 @@
 jobs:
-  complete_coverals:
-    needs: test
+  complete_coveralls:
+    needs: test-multi-python
     runs-on: ubuntu-latest
     steps:
-    - name: Coveralls Finished
-      uses: coverallsapp/github-action@57daa114ba54fd8e1c8563e8027325c0bf2f5e80
+    - uses: coverallsapp/github-action@57daa114ba54fd8e1c8563e8027325c0bf2f5e80
```

I am not a big fan of specifying commit like this.
Could we consider using :
```suggestion
    - uses: coverallsapp/github-action@master
```
to avoid technical debt

---

See this for the reasoning for for this specific commit: https://github.com/praw-dev/praw/pull/1402